doc/quick_tutorial.rst

```diff
@@ -26,35 +26,45 @@ Using OpenShot is very easy, and this tutorial will take you through the basics
 **under 5 minutes**. After this tutorial, you will be able to make a simple photo
 slide-show with music.
 
-Step 1 – Import Photos & Music
-------------------------------
-Before we can begin making a video, we need to import files into OpenShot. Drag and
-drop a few images (\*.JPG, \*.PNG, etc...) and a music file (most formats will work)
-from your Desktop to OpenShot. Be sure to drop the files where the
+Step 1 – Import media files
+---------------------------
+
+Before we can begin making a video, we need to import media files into OpenShot. Most video,
+image and music file formats will work. Drag and drop a few videos or images and a music file
+(most formats will work) from your Desktop to OpenShot. Be sure to drop the files where the
```

Comment:
This is a bit redundant in terms of "most formats will work", saying it twice in two sentences.

Reply:
Copy and paste leftover, fully agreed. Gone.